### ISSUE TITLE
Updating from version 0.10.0 to version 1.0.1 causes it to show the wrong pics or nothing

### ISSUE THREAD

**Issue by petercr** [timhagn/gatsby-background-image#109]
## Description
Hi Tim 👋  so when I updated my site from version 0.10.0 to version 1.0.1 it caused the site to totally fall apart 🧯 🔥 

What's happening is I have 3 different `<div>`s in a column. With the old version, each component query worked and showed the correct images with Base64 loading and all that jazz 🎷 

After I updated to version 1.0.1 the first 2 `<div>`s show nothing until you scroll down and the 3rd `<div>` becomes visible. Then all the backgrounds fill with the image from the 3rd query 🤷‍♂️ 
I get this same problem in both dev and production builds. Also I checked out all the GraphQL queries in GraphiQL, and opened their static addresses in the browser. This lead to the correct images, so it seemed strange. 

_And since_ there's no `CHANGELOG.md` file, I wondered if you could shed some light on this issue.
Or should I just go poking through all the commits and see if I can find what the heck changed 😬 

Thanks and if you want access to the repo just let me know, it's hosted on BitBucket. 

### Steps to reproduce

Updated `gatsby-background-image` from version 0.10.0 to version 1.0.1
Than ran `gatsby develop`

### Expected result
Gatsby Background component to show images that were show with the prior builds. It's been working since ~0.6.*

### Actual result
As describe above, none of the Gatsby Backround components show the image until the 3rd `<div>` is reached. Then they all show the image from the 3rd component. 

### Environment
It's Windows 10 running WSL2 Ubuntu. Error shows up in multiple browsers. 

```
System:
    OS: Linux 4.19 Ubuntu 18.04.4 LTS (Bionic Beaver)
    CPU: (6) x64 Intel(R) Core(TM) i5-8400 CPU @ 2.80GHz
    Shell: 4.4.20 - /bin/bash
  Binaries:
    Node: 12.16.1 - /usr/bin/node
    Yarn: 1.22.4 - /usr/bin/yarn
    npm: 6.14.4 - /usr/bin/npm
  Languages:
    Python: 2.7.17 - /usr/bin/python
  npmPackages:
    gatsby: ^2.20.15 => 2.20.15
    gatsby-background-image: ^1.0.1 => 1.0.1
    gatsby-image: ^2.3.2 => 2.3.2
    gatsby-plugin-google-analytics: ^2.2.2 => 2.2.2
    gatsby-plugin-manifest: ^2.3.3 => 2.3.3
    gatsby-plugin-offline: ^3.1.2 => 3.1.2
    gatsby-plugin-prefetch-google-fonts: ^1.4.3 => 1.4.3
    gatsby-plugin-react-helmet: ^3.2.1 => 3.2.1
    gatsby-plugin-sharp: ^2.5.4 => 2.5.4
    gatsby-plugin-sitemap: ^2.3.1 => 2.3.1
    gatsby-plugin-styled-components: ^3.2.1 => 3.2.1
    gatsby-source-filesystem: ^2.2.2 => 2.2.2
    gatsby-transformer-sharp: ^2.4.4 => 2.4.4
  npmGlobalPackages:
    gatsby-cli: 2.11.7
```

**Comment by timhagn** [comment]
Hi Peter,

but indeed, there is a [CHANGELOG](https://github.com/timhagn/gatsby-background-image/blob/master/packages/gatsby-background-image/CHANGELOG.md). Though that probably won't help with your issue, as I haven't changed a thing with the IO. Or were you depending on the deprecated classID ; )?

Best,

Tim.

**Comment by nik27** [comment]
I had the same problem and managed to fix it by adding different className to each BackgroundImage component. 

Problem is that when there is no className gatsby-background-image (or gatsby itself?) adds class .gatsby-background-image-wrapper to every BackgroundImage component and so the last item overrides previous ones.

**Comment by paulwhitaker** [comment]
I was experiencing the same issue. Fixed it using the work around posted by @nik27. Thanks for that, btw. It this the way you intended for this to work?

I added this to ensure that every component using this has a unique className, but it seem like a hack.

```javascript
className={`gatsby-background-image-${data.slug}`}
```

**Comment by timhagn** [comment]
Hi there, @petercr, @nik27 & @paulwhitaker,

thanks for such active participation, this was an easy one to fix. I only erroneously added `gatsby-background-image-wrapper` to prevent empty classes (& thus failing builds). Now it's getting a unique `className` when the prop is empty, as well (not only when `className` is already cached).

Hope it works for you and breaks no ones code (as this was only added superfluously in `v1.0.1` I didn't deem it necessary to escalate to 2.0 ; ): the just published `gbi(-es5)` `v1.1.0` : ).

When it fixed this issue, feel free to close it, @petercr!

Best,

Tim.

**Comment by petercr** [comment]
Thanks Tim 😄 version `1.1.1` _**totally**_ fixes the problem 👍 
I will go ahead and close the issue. 
Another 🐛 stomped out by, with ⚡ speed
